mobile/src/modules/transactions/components/send/SendBurstForm.tsx

```diff
@@ -43,7 +54,12 @@ export class SendBurstForm extends React.PureComponent<Props, SendBurstFormState
     sender: null,
     address: this.props.deepLinkProps && this.props.deepLinkProps.address || 'BURST-',
     amount: this.props.deepLinkProps && this.props.deepLinkProps.amount || '',
-    fee: this.props.deepLinkProps && this.props.deepLinkProps.fee || ''
+    fee: this.props.deepLinkProps && this.props.deepLinkProps.fee ||
+    this.props.suggestedFees && convertNQTStringToNumber(this.props.suggestedFees.standard.toString()).toString() || '',
+    message: this.props.deepLinkProps && this.props.deepLinkProps.message || undefined,
```

Comment:
I would use a small utility func: 
`const get = (base, propName, default) => base && base[propName] || default`

Reply:
Just patiently waiting for optional chaining :)